runner/full.go

```diff
@@ -83,33 +132,32 @@ func processDatabase(server state.Server, globalCollectionOpts state.CollectionO
 			if server.Grant.Valid {
 				logger.PrintVerbose("Could not acquire snapshot grant, reusing previous grant: %s", err)
 			} else {
-				return state.PersistedState{}, state.Grant{}, err
+				return state.PersistedState{}, state.Grant{}, state.CollectionStatus{}, err
 			}
 		} else {
 			server.Grant = newGrant
 		}
 	}
 
-	transientState := state.TransientState{}
```

Comment:
There doesn't seem to be a good reason to actually have this state around? Or am I missing something?

Reply:
Yeah I think this was a leftover from a prior iteration where the sentry client was passed around (which is no longer the case)